src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -7198,15 +7215,31 @@ namespace InternalHeatGains {
             state.dataHeatBal->spaceIntGain(spaceNum).QOCTOT += state.dataHeatBal->People(Loop).TotGainRate;
         }
 
+        for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
```

Comment:
Attempt at a small efficiency gain by setting zone-level gain components by summing the space-level components instead of summing both piece-by-piece.  Just for people, here, could do the same for other components, but it's only helpful when there are multiple objects of the same type within a space(s).